plutus-core/plutus-ir/test/type-errors/typeLet.golden

```diff
@@ -1,2 +1,7 @@
-E019: Error during PIR typechecking:
-Free type variable at  typeLet:4:8 :  a
\ No newline at end of file
+E016: Error during PIR typechecking:
+Type mismatch at typeLet:4:5 in term
+  '(con integer 1)'.
+Expected type
```

Comment:
Hmm, I think these expected and actual reports are the wrong way around, no? We should be expecting `a` and find `integer`. I wonder if one of the error cases is passing the expected/actual things the wrong way around. Something to look into after this perhaps.